docs/core/introduction.md

```diff
@@ -18,7 +18,7 @@ ms.custom: "updateeachrelease"
   * [Windows Forms](/dotnet/desktop/winforms/)
   * [Universal Windows Platform (UWP)](/windows/uwp/get-started/create-a-hello-world-app-xaml-universal)
 * [Games](https://dotnet.microsoft.com/apps/games)
-* [Internet of Things (IoT)](/dotnet/iot)
+* [Internet of Things (IoT)](../iot/index.yml)
```

Comment:
The iot folder is not in this repo, so I don't think this is a valid link.

Reply:
It is in this repo (https://github.com/dotnet/docs/blob/master/docs/iot/index.yml).  Otherwise the build would have failed on that link since it's file relative.